src/sources/host_metrics.rs

```diff
@@ -101,7 +101,7 @@ impl Namespace {
 }
 
 #[derive(Clone, Debug, Default, Deserialize, Serialize)]
-struct HostMetricsConfig {
+pub struct HostMetricsConfig {
```

Comment:
I imagine these would be better suited as `pub(crate)`, but that's not a blocking comment :)